modules/api-svc/src/lib/dto.js

```diff
@@ -0,0 +1,29 @@
+const { Directions, SDKStateEnum} = require('./model/common');
+
+const quoteRequestStateDto = (request) => ({
+    // transferId: this follows the slightly dodgy assumption that transferId will be same as this transactionId.
+    // so far this has held in moja implementations but may not always be the case. regardless, future FSPIOP API
+    // versions MUST deal with this cleanly so we can expect to eliminate this assumption at some point.
+    transferId: request.body.transactionId,
+    direction: Directions.INBOUND,
+    quoteRequest: {
+        headers: request.headers,
+        body: request.body
+    },
+    currentState: SDKStateEnum.QUOTE_REQUEST_RECEIVED,
+    initiatedTimestamp: new Date().toISOString(),
+});
+
+const fxQuoteRequestStateDto = (request) => ({
+    // assume commitRequestId from fxTransfer should be same as conversionTerms.conversionId from fxQuotes
+    conversionId: request.body.conversionTerms.conversionId,
+    fxQuoteRequest: request,
```

Comment:
Is it ok to pass the whole request object here instead of following quote implementation? I think its better to follow the same (passing only headers and body).

Reply:
but `request` contains only _body_ and _headers_  (first param to _model.postFxQuotes()_ method):
https://github.com/mojaloop/sdk-scheme-adapter/blob/feat/fx-impl-3618/modules/api-svc/src/InboundServer/handlers.js#L965